### ISSUE TITLE
Custom fonts don't appear consistently on Safari or Chrome

### ISSUE THREAD

**Issue by slambert** [thec4aa/c4aa-org#307]
**Describe the bug**

If you load the page in Firefox, all the custom fonts appear.
If you load the page in Safari or Chrome, it seems WorkSans does not load. ZillaSlab does appear to load

In the current (2024-04-30) develop branch - looking at https://backtobasics.c4aa.org

**To Reproduce**
1. Go to https://backtobasics.c4aa.org/about
2. See error

**Expected behavior**
Fonts would appear consistently across browsers

**Screenshots**
Firefox:

![Screenshot 2024-04-30 at 8 43 55 AM](https://github.com/thec4aa/c4aa-org/assets/67271/d53cab7d-3481-4151-8715-1a42f2069d45)

Safari:

![Screenshot 2024-04-30 at 8 44 25 AM](https://github.com/thec4aa/c4aa-org/assets/67271/27aa3875-7b18-4028-913d-191e95711041)

**Desktop (please complete the following information):**
 - OS: macos
 - Browser FF and Safari
 - Safari 17.4.1, FF 125.0.2

**Smartphone (please complete the following information):**
Also seen on smart phone

**Additional context**

**Comment by slambert** [comment]
@ashokmladumor I noticed what I did! Included "Regular" when it should have just been "Work Sans"  🤦

Was:
`"fontFamily": "\"Work Sans Regular\", Helvetica Neue, Helvetica, Arial, sans-serif",`

Now:
`"fontFamily": "\"Work Sans\", Helvetica Neue, Helvetica, Arial, sans-serif",`

Just made a commit. The font weight is not working yet though.

I had been using this page as a reference: https://cr0ybot.com/2023/07/using-variable-fonts-in-theme-json/
